core/vulkan/cc/include/vulkan/vulkan_fuchsia.h

```diff
@@ -1,10 +1,6 @@
 #ifndef VULKAN_FUCHSIA_H_
 #define VULKAN_FUCHSIA_H_ 1
 
-#ifdef __cplusplus
-extern "C" {
-#endif
-
 /*
```

Comment:
Should not the license be at the top of the file or does it not matter?

Reply:
This is pulled directly from the vulkan repo, I don't want to change it much.